lib/openstudio-standards/standards/necb/NECB2011/building_envelope.rb

```diff
@@ -776,6 +776,7 @@ def apply_max_srr_nrcan(model:, srr_lim:, srr_opt: '')
         sub_surface_create_scaled_subsurfaces_from_surface(surface: roof, area_fraction: srr_lim, construction: skylight_construct_set)
       end
     else # OPTION B
+      TBD.clean!
```

Comment:
_OSut_ piggy-backs on top of _TBD_. It shares the same internal logger. Clean reset.